client/src/store/modules/user.js

```diff
@@ -43,7 +48,8 @@ export default {
         const user = await firebase
           .auth()
           .signInWithEmailAndPassword(payload.email, payload.password);
-        const userPayload = { id: user.user.uid };
+        const username = user.user.displayName;
+        const userPayload = { id: user.user.uid, username: username };
```

Comment:
I'd rather have it like this: 
{ 
id: user.user.uid,
username: user.user.displayName
 }
for consistency

Reply:
sure